easy_rec/python/input/odps_input_v2.py

```diff
@@ -8,7 +8,7 @@
 from easy_rec.python.utils import odps_util
 
 try:
-  import pai
+  from pai.data import WorkQueue
```

Comment:
does it work both on pai-tf 1.12 and deeprec(pai-tf 1.15?)

Reply:
deeprec should use `from tensorflow.python.ops.work_queue import WorkQueue`

Reply:
Done.